atc/metric/emitter/prometheus.go

```diff
@@ -391,6 +393,16 @@ func (config *PrometheusConfig) NewEmitter() (metric.Emitter, error) {
 	)
 	prometheus.MustRegister(checksEnqueued)
 
+	volumesStreamed := prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Namespace: "concourse",
+			Subsystem: "volumes",
```

Comment:
```suggestion
			Subsystem: "workers",
```

Reply:
@xtreme-sameer-vohra I suppose `workers` subsystem counters should all have a worker associated, but `volumes streamed` is actually an ACT metrics.